src/modules/handletodo.js

```diff
@@ -0,0 +1,66 @@
+import Todo from './todolist.js';
+import Task from './list.js';
+
+const container = document.createElement('div');
+const input = document.createElement('input');
+const ul = document.createElement('ul');
+const clearBtn = document.createElement('button');
+// toDoLists data
+let toDoLists = [];
+
+// Main component that holds all the elements
+export const render = () => {
+  toDoLists = JSON.parse(localStorage.getItem('toDoLists')) || [];
+  // Adding classes and attributes to DOM elements
+  container.classList.add('list-container');
+  input.classList.add('task-desc');
+  input.placeholder = 'Add to your list...';
+  clearBtn.textContent = 'Clear all completed';
+  clearBtn.classList.add('clear-btn');
+  clearBtn.disabled = true;
+  container.append(input);
+  // Mapping through toDoLists array
+  ul.innerHTML = '';
+  toDoLists.map((toDoList) => {
+    ul.appendChild(Todo(toDoList));
+    return 'completed';
+  });
+  container.append(ul);
+  // Appending DOM elements to parent container
+  container.append(clearBtn);
+  const mainContainer = document.querySelector('.main');
+  mainContainer.appendChild(container);
+};
+
+const addTostorage = () => {
+  localStorage.setItem('toDoLists', JSON.stringify(toDoLists));
+};
+// adding tasks to the todolist array
+export const addTask = (description) => {
+  const newTask = new Task();
+  newTask.description = description;
+  newTask.completed = false;
+  newTask.taskindex = toDoLists.length + 1;
+  toDoLists.push(newTask);
+  addTostorage();
+  window.location.reload();
+};
+// adding to local storage
+
+export const editTask = (newDescription, editindex) => {
+  toDoLists[editindex - 1].description = newDescription;
+  addTostorage();
+};
+
+export const removeTask = (delteIndex) => {
+  // remove object from array
+  const newindex = Number(delteIndex);
+  toDoLists.splice(newindex - 1, 1);
+  // reset indexes of all array values
+  const maxIndex = toDoLists.length;
+  for (let i = 0; i < maxIndex; i += 1) {
+    toDoLists[i].taskindex = i;
+  }
+  addTostorage();
+  window.location.reload();
+};
```

Comment:
According to the project requirements, deleting a task should update all remaining items' indexes, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1). This functionality has not been correctly implemented as the indexes of the remaining tasks start from zero as seen in the video below:


https://github.com/demesameneshoa/to-do-list/assets/37116322/b80279f8-ced6-44ff-bca4-183a1f5b7e23



Kindly update your app so the delete functionality is implemented correctly🙏